apps/calendar/js/views/modify_account.js

```diff
@@ -8,7 +8,6 @@ var Presets = require('presets');
 var URI = require('utils/uri');
 var View = require('view');
 
-require('css!modify_account_view');
```

Comment:
I decided to bundle all the lazy loaded styles into a single stylesheet since that was faster/simpler.